csrc/attention/attention_kernels.cu

```diff
@@ -22,12 +22,26 @@
 
 #include "attention_dtypes.h"
 #include "attention_utils.cuh"
-#ifdef ENABLE_FP8_E5M2
+
+#if defined(ENABLE_FP8_E5M2)
 #include "../quantization/fp8_e5m2_kvcache/quant_utils.cuh"
+#elif defined(ENABLE_FP8_E4M3)
+#include "../quantization/fp8/amd_detail/quant_utils.cuh"
```

Comment:
I thought E4M3 is NV compatible as well? Also is it possible to enable both?

Reply:
@simon-mo , OCP E4M3 is NV compatible. We don't address NV platform in this PR.